### ISSUE TITLE
Does StrongLoop/IBM still provide translations?

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/expressjs.com#1647]
As the title says, does StrongLoop/IBM still provide the translations mentioned on the page?

**Comment by UlisesGascon** [comment]
No AFAIK

**Comment by crandmck** [comment]
Well, StrongLoop no longer exists (and hasn't for many years, since it was acquired by IBM) and IBM no longer provides any translation support.

I assume you're referring to this statement on the website home page:
> Documentation translations provided by StrongLoop/IBM

This translated content WAS originally provided by IBM/StrongLoop, as you can see if you look at many of the l10n files.  Some have been updated, but some have not changed in 9 years when the translations were originally provided.   

So as it stands, this is an accurate statement.  If it seems confusing, we could try to clarify it.

**Comment by bjohansebas** [comment]
It seems confusing to me because it gives the impression that they are still providing the translations

**Comment by chrisdel101** [comment]
So regarding translations, if we add a new page to the English site, does this then need to be re-translated into every language by humans?
I'm assuming yes, but just want to make sure I get it.

**Comment by bjohansebas** [comment]
Yes, if a new page is added, ideally it should also be added to the other languages, even if they aren’t translated yet. However, I think this addition should be done in a separate PR

**Comment by ShubhamOulkar** [comment]
Now Crowdin provide translations.

**Comment by bjohansebas** [comment]
I was waiting to merge (https://github.com/expressjs/expressjs.com/pull/1779) to close this.
